autoscheduler/frontend/src/tests/ui/SelectTerm.test.tsx

```diff
@@ -12,60 +18,85 @@ jest.mock('@reach/router', () => ({
   navigate: jest.fn(),
 }));
 
+// Mocks the fetch call to api/terms
+const mockTermsAPI = (): void => {
+  fetchMock.mockResponseOnce(JSON.stringify({
+    'Fall 2020': '202031',
+    'Summer 2020': '202021',
+    'Spring 2020': '202011',
+    'Fall 2019': '201931',
+    'Summer 2019': '201921',
+    'Spring 2019': '201911',
+  }));
+};
+
+
 describe('SelectTerm', () => {
+  beforeEach(mockTermsAPI);
+
+  afterEach(fetchMock.mockRestore);
+
   describe('Menu opens', () => {
-    test('Menu opens after button is clicked', () => {
+    test('Menu opens after button is clicked', async () => {
       // arrange
       const store = createStore(autoSchedulerReducer);
 
-      const { getByText } = render(
-        <Provider store={store}>
-          <SelectTerm />
-        </Provider>,
-      );
+      let getByText: Function;
+      await act(async () => {
+        ({ getByText } = render(
+          <Provider store={store}>
+            <SelectTerm />
+          </Provider>,
+        ));
+      });
 
       // act
       const button = getByText('Select Term');
-      fireEvent.click(button);
+      act(() => { fireEvent.click(button); });
```

Comment:
Yeah, I did that because I saw other test wrap `fireEvent` in act (and I was getting warnings about changing state outside of `act`) so I had assumed it was still necessary, but the warnings were only coming from the `render` calls